docs/dg/dev/sdks/sdk/development-tools/phpstorm-support.md

```diff
@@ -0,0 +1,49 @@
+---
+title: PHPStorm Support
+description: Enable Spryker tools in PHPStorm to enhance your development experience
+last_updated: Oct 20, 2024
+template: concept-topic-template
+redirect_from:
+  - /docs/scos/dev/sdk/development-tools/development-tools.html
+related:
+  - title: Code sniffer
+    link: docs/scos/dev/sdk/development-tools/code-sniffer.html
+  - title: PHPStan
+    link: docs/scos/dev/sdk/development-tools/phpstan.html
+
+---
+
+## PHPStorm Support
+Spryker provides support for tools that perform code style fixes, static analysis, and architecture checks, each accessible within PHPStorm’s External Tools panel. These tools are available at both module and project levels and can be executed directly by right-clicking within the project directory tree.
+
+The following tools are included:
+- `console code:sniff:style -f`: Applies coding style fixes (module folders only).
+- `console code:phpstan`: Runs static analysis (module folders only).
+- `console code:sniff:architecture`: Detects architecture rule violations (module folders only).
+
+### Installation
+The installed commands within PHPStorm can be registered to run either as local machine commands or Spryker Docker SDK commands, depending on your configuration choice during installation. Consequently, ensure the appropriate environment is available (e.g., Spryker Docker SDK running for `docker` platform commands, or required tools installed locally for `local` platform commands).
+
+The script currently supports Mac systems only.
+
+**Prerequisite**:
+Ensure that `awk` is installed on your local machine, as it is required to run the commands directly from PHPStorm.
+
+To install the Spryker development tools configuration in PHPStorm:
```

Comment:
Does this script actually install something or just adds configuration?

Reply:
Hmm, i'm not sure if i know what is the difference. It adds configuration only, but this adding is basically installs/enables functionality in PhpStorm.